docs/operations-guide/running-metabase-on-azure.md

```diff
@@ -0,0 +1,183 @@
+
+![Azure VNET integration](images/AZVNETintegration.png)
+
+Now click on the huge plus sign next to **Add VNET** and select the VNET that you created on the first step and the public subnet to set the server on the public subnet of the VNET, and click OK.
+
+![Azure VNET public subnet](images/AZVNETPublicSubnet.png)
+
+Now go back to te application configuration page and click on Settings --> Configuration on the left side of the page where you will see a few Application Settings already configure.
+
+Here you will need to add the [Enviroment Variables]() for connecting Metabase to its [PostgreSQL Application Database](), but make sure that you use the **MB_DB_CONNECTION_URI** as PostgreSQL in Azure is configured by default with SSL so currently the only way of passing those parameters is by using the full URI.
+
+Also, take into account that the username in Azure PostgreSQL is `user@name_of_your_database_engine` so in this case the entire connection uri would be as follows: `postgresql://databasePrivateIPAddress:port/postgres?user=user@name_of_your_database_engine&password=configuredpassword&ssl=true&sslmode=required`.
+
+Lets put an example, if you got this information from the previous steps:
+1) database private IP address: 10.0.2.4
+2) database port: 5432 (in the case of Postgres, MySQL/MariaDB default port is 3306)
+3) name of the database server: metabase-app-database
+4) username of the database: metabase
+5) password: Password1!
+
+then my connection string will be: `postgresql://10.0.2.4:5432/postgres?user=metabase@metabase-app-database&password=Password1!&ssl=true&sslmode=require`
+
+Click on save and the instance will restart. After this step you will be able to access your Metabase in the URL that you have in the "Overview" tab in the web app under the URL section (will be the name you chose in the first step of the config of the web app + .azurewebsites.net as Azure handles all the deployment and domain)
+
+## Additional configurations
+### How to enable Health checks
+
+Enabling health checking in Metabase is a good practice. You can tell the "host" when Metabase is ready to server your customers by just doing a simple configuration: go to your web app -> Monitoring -> Health Check -> Enable health check and include in the path `/api/health`.
+
+### How to update
+
+Simply go to your web app and in Settings -> Container Settings (Classic) and there change the image version of Metabase.
+In the same way you do an upgrade, you can change the Metabase version to Enterprise by switching to v1 instead of v0 in the prefix of the docker image.
+
+**Important**: always ensure you have a backup of your Database before doing a version upgrade, *specially*, when upgrading between major versions.
+
+### How to see the logs
+
+If you want to see the logs of your deployment, you need to go to the web app that you created and you will be able to see the streaming logs in Monitoring -> Log stream.
```

Comment:
```suggestion
Visit your web app in Azure, and navigate to **Monitoring -> Log stream**.
```
Do we want to point them to the troubleshooting logs in the Metabase Admin panel?